Scenario Based Samples/Iframe Full Server Embedding/README.md

```diff
@@ -0,0 +1,88 @@
+   
+    ![JWT Authentication](https://github.com/boldbi/samples/assets/129487075/c7d429ee-eb6e-4d3f-aa49-0e9d9bddef91)
+ 
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/37f439d6-3f58-43b3-807f-b90aeb3e2fbe)
+
+ * Open the `UMS Site Settings` page in Bold BI Server.  In `Authentication` section, select the `General` tab and enable `Enable Default Authentication` by selecting `JWT SSO` option in the `Default Authentication Provider` dropdown and `Save` the changes.
+ 
+    ![Default Authentication](https://github.com/boldbi/samples/assets/129487075/a81894ac-c147-41df-8c97-ed9928d16953)
+    
+ * Click on `Configuration` in UMS Settings, then select the `config.xml` file from the dropdown.  Change the value of the `<EnableSameTabLinkTarget>` tag from `false` to `true` in order to render the URL within the same application and `Save` the changes.
+    
+    ![Configuration](https://github.com/boldbi/samples/assets/129487075/b3d65c3c-d470-4f53-9983-a35291bb6bfe)
+
+ * Open the Administration Site Settings page in Bold BI Server. Under `Authentication`, select the `General` tab and enable `Enable Default Authentication`, then `Save` the changes. (ex: http://localhost:50000/bi/site/site1/administration)
+    
+    ![Admin Authentication](https://github.com/boldbi/samples/assets/129487075/763ae9a6-4448-40c3-9897-afb28db745ed)
+
+ * In the application, change the `jwt:boldbiserverurl` value in the `appsettings.json` file to the URL of our Bold BI server.
+  
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/efb08c80-ffcc-453e-b216-0949b06b9126)
+
+    
+ ## Run a Sample Using Command Prompt 
+    
+  1. Open the command prompt and navigate to the specified file [location](https://github.com/boldbi/samples/tree/master/Scenario%20Based%20Samples/Iframe%20Full%20Server%20Embedding/IframeFullServer) where the project is located.
+
+  2. Execute the command `dotnet restore` to restore the necessary packages. Once the packages have been successfully restored, use the `dotnet build` command to build the project.
+  
+  3. Finally, run the application using the command `dotnet run`.  It will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
+
+ ## Developer IDE
+
+  * [Visual studio code](https://code.visualstudio.com/download)
+  
+### Run a Sample Using Visual Studio Code
+ 
+  * Open the folder `IframeFullServer` in Visual Studio Code.
+  
+  * Open the terminal in Visual Studio Code and execute the command `dotnet restore` to restore the required dependencies.
+ 
+  * Build your .NET project by executing the `dotnet build` command in the terminal.
+ 
+  * To run the application, use the command `dotnet run` in the terminal. After executing the command, the application has started and it will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
```

Comment:
(e.g., http://localhost:4000) 

- URL port 4 digits only.

Reply:
Copy the URL and paste it into your web browser.

Reply:
updated